backend/route.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, settings):
             (r"/elixir/logout",                                                       auth.ElixirLogoutHandler),
             (r"/google/login",                                                        auth.GoogleLoginHandler),
             (r"/google/logout",                                                       auth.GoogleLogoutHandler),
+            (r"/transfer",                                                            auth.UpdateUserHandler),
```

Comment:
This is an api endpoint, should probably be at `/api/user/transfer` or maybe `/api/user/elixir_transfer`.